src/components/TopNav.js

```diff
@@ -119,6 +125,14 @@ function TopNav({ isMaintenance, loggedIn = false }) {
               </NavItem>
             </>
           )}
+          {!config.is_local_studio && (
+            <li className="nav-item">
+              <a target="_blank" rel="noreferrer" href="https://harperdb.io/docs" id="viewResources" title="HarperDB Documentation">
+                <i className="fas fa-tools" />
+                <span className="d-none d-lg-inline-block">&nbsp;Resources</span>
```

Comment:
Instead of using whitespace, I'd prefer to use some css like padding or margin to properly space the icon and text.

Reply:
Fixed!